script/add-codeql-to-path

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+
+cd "$(dirname "$0")/.."
+
+gh extensions install github/gh-codeql
+gh codeql download
```

Comment:
Ideally, I want to check whether `gh codeql` is already installed/downloaded, and skip this step if so...

Reply:
I just tried it and it looks like the command knows to check this so you don't need to worry about that.